src/mlpack/methods/hoeffding_trees/hoeffding_tree_impl.hpp

```diff
@@ -341,7 +452,7 @@ void HoeffdingTree<
     delete dimensionMappings;
 
   const CategoricalSplitType<FitnessFunction> categoricalSplitIn(0, 0);
-  const NumericSplitType<FitnessFunction>& numericSplitIn(0);
+  const NumericSplitType<FitnessFunction> numericSplitIn(0);
```

Comment:
Wow, did this even compile before?  Nice fix!

Reply:
Thanks haha!